src/Build/Logging/SimpleErrorLogger.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.Logging.SimpleErrorLogger
+{
+    /// <summary>
+    /// This logger ignores all message-level output, writing errors and warnings to
+    /// standard error, colored red and yellow respectively.
+    ///
+    /// It is currently used only when the user requests information about specific
+    /// properties, items, or target results. In that case, we write the desired output
+    /// to standard out, but we do not want it polluted with any other kinds of information.
+    /// Users still might want diagnostic information if something goes wrong, so still
+    /// output that as necessary.
```

Comment:
What is the motivation for introducig new single purpose logger for this, while leaving the existing `ConsoleLogger` have the unfixed behavior?
Shil this be actualy made part of the existing `ConsoleLogger`?

Reply:
Oh I see - this is intended for the https://github.com/dotnet/msbuild/pull/8792 that already uses the logger. That's a bit confusing. If you want to have 2 separate PRs (I understand that) - how about making this PR push into that branch then (you'd need to first push it into this repo though, so that such PR occurs here)?

In any case - do we actually still want to fix the same thing for the other loggers that uses the VT? Otherwise this PR probably should not be marked as fixing https://github.com/dotnet/msbuild/issues/9079